share/lib/python/scripts/_binwrapper.py

```diff
@@ -43,4 +46,19 @@ def _config_exe(exe_name):
 
 if __name__ == '__main__':
     exe = _config_exe(os.path.basename(sys.argv[0]))
-    os.execv(exe, sys.argv)
+
+    # special is a now a full-on binary. therefore, we've wrapped around special
+    # in order to still be able to configure NEURON environment variables (run above)
+    if exe.endswith('special'):
+        # point executable to `special.nrn`
+        exe = os.path.join(sys.argv[0]+'.nrn')
+
+    subprocess.call([exe, *sys.argv[1:]])
```

Comment:
@alexsavulescu : why `subprocess.call` instead of `os.execv` ? IIRC, later was "replacing" the executing process, which we wanted.

Reply:
Ah, because `execv` returns and the wrapping wouldn't be reached. We could adjust the logic then